app/src/scenes/inscription/Create/stepDone.js

```diff
@@ -27,8 +31,101 @@ export default function StepDone() {
       <Info>
         <h3>INSCRIPTION AU SNU</h3>
         <h1>Vous y êtes presque !</h1>
-        <p>Vous êtes sur le point de soumettre votre dossier à l&apos;administration du SNU.</p>
-
+        <p>
+          Vous êtes sur le point de soumettre votre dossier à l&apos;administration du SNU. Veuillez vérifier vos informations avant de valider votre demande d&apos;inscription.
+        </p>
+        <InfosContainer>
+          <Bloc title="Mon profil" profile>
+            <Details title="Nom" value={young.lastName} />
+            <Details title="Prénom" value={young.firstName} />
+            <Details title="Sexe" value={young.gender} />
```

Comment:
il faudrait traduire ce champs.
je sais pas si on le fait la ou si on essaie de traduire tout le temps la value dans `<Details />`, as you wish